main_routers/config_router.py

```
@@ -0,0 +1,354 @@
+        logger.error(f"获取 Steam 语言设置失败: {e}")
+        return {
+            "success": False,
+            "error": str(e),
+            "steam_language": None,
+            "i18n_language": None
+        }
+
+
+@router.get("/config/core_api")
+async def get_core_config_api():
+    """获取核心配置（API Key）"""
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)